great-idea/css/index.css

```diff
@@ -62,3 +62,129 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+.container {
+	width: 1100px;
+	
+}
+
+header {
+	display: flex;
+	justify-content: space-between;
+	width: 1100px;
+	margin-bottom: 40px;
+}
+
+header nav {
+	width: 700px;
+	padding: 30px;
+	display: flex;
+	justify-content: space-evenly;
+
+}
+
+header nav a {
+	color: gray;
+	text-decoration: none;
+	display: flex;
+	
+}
+
+header .logo {
+	display: flex;
+	padding: 20px;
+}
+
+header .logo2 {
+	display: flex;
+}
+
+.cta {
+	width: 1000px;
+	display: flex;
+	align-items: center;
+	justify-content: space-around;
+	padding-bottom: 50px;
+}
+
+.cta .cta-text {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	align-items: center;
+	padding: 0px 100px;
+}
+
+.cta .cta-text h1 {
+	font-size: 80px;
+	text-align: center;
+}
+
+.cta .cta-text button {
+	background-color: white;
+	font-family: 'Titillium Web';
+	font-size: 18px;
+	padding: 3px 40px;
+	border-style: solid;
+	border-color: black;
+	border-width: 1px;
+}
+
+.main-content {
+	width: 1050px;
+	margin-left: 30px;
+	margin-top: 30px;
+	margin-bottom: 50px;
+	padding: 50px 0px;
+	display: flex;
+	flex-direction: column;
+	align-content: space-between;
+	justify-content: center;
+	border-top: 1px solid black;
```

Comment:
Great implementation of Flexbox using multiple functions together, and a good understanding of how weight will effect the children of this element.